github/ci/prow/files/labels.yaml

```diff
@@ -428,3 +428,56 @@ repos:
         target: prs
         prowPlugin: approve
         addedBy: approvers
+  nmstate/kubernetes-nmstate:
+    labels:
+      - name: priority/highest
```

Comment:
This is a label that exists in a different form in the [target repo](https://github.com/nmstate/kubernetes-nmstate/labels) where it's `priority:highest` instead of `priority/highest`. Maybe you should rename your labels before applying this?

Reply:
I mean, you should either change them in the target repo if that is an option (I think that GitHub support renaming labels and keeps the issues with the updated labels). Or you should change them here.

Reply:
Super good point, have renamed the labels to the ones we have at this PR so they track same issues/prs